subcommand/server-acl-init/command.go

```diff
@@ -491,6 +497,60 @@ func (c *Command) Run(args []string) int {
 		}
 	}
 
+	if len(c.flagIngressGateways) > 0 {
+		// Create a token for each ingress gateway name
+		for _, gateway := range c.flagIngressGateways {
+			// Check that the gateway name is not an empty string
+			if gateway == "" {
+				c.log.Error("Ingress gateway names cannot be empty")
+				return 1
+			}
+
+			// Define the gateway rules
+			ingressGatewayRules, err := c.ingressGatewayRules(gateway)
+			if err != nil {
+				c.log.Error("Error templating ingress gateway rules", "gateway name", gateway, "err", err)
```

Comment:
```suggestion
				c.log.Error("Error templating ingress gateway rules", "gateway-name", gateway, "err", err)
```

I think we don't use spaces in these keys (could be wrong).